wpilib/wpilib/drive/differentialdrive.py

```diff
@@ -70,13 +101,13 @@ class DifferentialDrive(RobotDriveBase):
     def __init__(self, leftMotor, rightMotor):
         """Constructor for DifferentialDrive.
 
-        .. note:: This class takes in a SpeedController per side. For two and four motor drivetrains,
-                  construct and pass in  :class:`.SpeedControllerGroup` instances as follows.
+        .. note:: To pass multiple motors per side, use a :class:`.SpeedControllerGroup`.
+                  If a motor needs to be inverted, do so before passing it in.
 
-        :param leftMotor: Left motor SpeedController
-        :param rightMotor: Right motor SpeedController
-        :type leftMotor: :class:`.SpeedController` or :class:`.SpeedControllerGroup`
-        :type rightMotor: :class:`.SpeedController` or :class:`.SpeedControllerGroup`
+        :param leftMotor: Left motor(s)
+        :param rightMotor: Right motor(s)
+        :type leftMotor: :class:`.SpeedController`
```

Comment:
Why get rid of the or? I guess a SpeedControllerGroup is a speed controller...But it makes it clearer in the docs imo

Reply:
It also made the docs not that pretty. Given there's a note right above it, I don't think it's very useful nor practical to list it again here.